[

{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#CVE-2018-14614", "@type": ["http://www.w3.org/2002/07/owl#NamedIndividual", "http://docs.oasis-open.org/cti/ns/stix#Vulnerability"], "http://docs.oasis-open.org/cti/ns/stix/core#description": [{"@value": "An issue was discovered in the Linux kernel through 4.17.10. There is an out-of-bounds access in __remove_dirty_segment() in fs/f2fs/segment.c when mounting an f2fs image."}], "http://docs.oasis-open.org/cti/ns/stix/core#modified": [{"@value": "2021-03-13T20:12:58.102407Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#created": [{"@value": "2021-03-13T20:12:58.102407Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#spec_version": [{"@value": "2.1"}], "http://docs.oasis-open.org/cti/ns/stix/core#id": [{"@value": "vulnerability--c44035b6-3b6f-4fe9-8ccb-e4a69c166940"}], "http://docs.oasis-open.org/cti/ns/stix/core#name": [{"@value": "CVE-2018-14614"}], "http://docs.oasis-open.org/cti/ns/stix/core#created_by_ref": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a"}], "http://docs.oasis-open.org/cti/ns/stix/marking#object_marking_refs": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"}]}

]